src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -436,6 +438,7 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 
 		if (isResponseVM(element)) {
 			this.codeBlocksByResponseId.set(element.id, codeblocks);
+			result.element.ariaLabel = this._accessibilityProvider.getAriaLabel(element);
```

Comment:
This will be some random subelement in the list item and probably isn't being read? You have two accessibility providers now and should only have one that is registered with the list